shell/server/pre-meteor.js

```diff
@@ -375,11 +376,71 @@ Meteor.startup(function () {
     WebApp.rawConnectHandlers.use(BlackrockPayments.makeConnectHandler(globalDb));
   }
 
-  WebApp.rawConnectHandlers.use(function (req, res, next) {
+  // This function serves responses on Sandstorm's main HTTP/HTTPS
+  // port.
+  var serveMeteorOrStaticPublishing = function(req, res, next) {
+    return dispatchToMeteorOrStaticPublishing(req, res, next, false, true);
+  }
+
+  // Bind listeners to FD #4 and higher, if we are supposed to be
+  // listening on multiple ports.
+  function getNumberOfAlternatePorts() {
+    var numCommas = (process.env.PORT.match(/,/g) || {}).length;
```

Comment:
Also maybe it would be easier to say: `var numPorts = process.env.PORT.split(",").length`